File: tools/excel/src/test/java/top/bettercode/summer/tools/java/PoiExcelTest.java
package top.bettercode.summer.tools.java;

import org.jetbrains.annotations.NotNull;
import org.junit.jupiter.api.Test;
import top.bettercode.summer.tools.excel.write.ExcelWriter;

/**
 * @author Peter Wu
 */
public class PoiExcelTest extends ExcelTest {

  public PoiExcelTest() {
    nameSuff = "2";
  }

  @NotNull
  @Override
  protected ExcelWriter getExcelWriter(String filename) {
    return ExcelWriter.of(filename, true);
  }

  @Test
  @Override
  public void testExport() {
    super.testExport();
  }

  @Test
  @Override
  public void testMergeExport() {
    super.testMergeExport();
  }

  @Test
  @Override
  public void testTemplate() {
    super.testTemplate();
  }
}

File: tools/excel/src/test/kotlin/top/bettercode/summer/tools/excel/PoiExcelTest.kt
package top.bettercode.summer.tools.excel

import org.junit.jupiter.api.Test
import top.bettercode.summer.tools.excel.write.ExcelWriter

/**
 *
 * @author Peter Wu
 */
class PoiExcelTest : ExcelTest() {
    override var nameSuff: String = "2"

    override fun excelExport(filename: String): ExcelWriter {
        return ExcelWriter.of(filename, true)
    }

    @Test
    override fun testExport() {
        super.testExport()
    }

    @Test
    override fun testMergeExport() {
        super.testMergeExport()
    }

    @Test
    override fun testTemplate() {
        super.testTemplate()
    }
}